client/my-sites/term-tree-selector/add-term.jsx

```diff
@@ -220,7 +220,7 @@ export default connect(
 	( state, ownProps ) => {
 		const { taxonomy, postType } = ownProps;
 		const siteId = getSelectedSiteId( state );
-		const { labels } = getPostTypeTaxonomy( state, siteId, postType, taxonomy ) || {};
+		const { labels } = getPostTypeTaxonomy( state, siteId, postType, taxonomy ) || { labels: {} };
```

Comment:
When selecting a draft from the drafts drawer a `null` exception was happening due to `labels` being undefined.